### PR TITLE
refactor(tip, tip-group, tip-manager): relocate base :host styles, style changes to match figma

### PR DESCRIPTION
**Pull request by rmstinson**
**Related Issue:** #2471 #2469

## Summary
Relocated the base :host styles for the component. Made style changes for the tip-manager to match style in Figma!

### CODE REVIEW

**Comment by bstifle** [comment]
@rmstinson yo!

so, theres not enough in the demo page to give this an accurate review.

Can you add tip, tip-group and some iterations of those?

**Comment by bstifle** [comment]
image
1. tip-group's padding and close X hit state is a little wonky. can we get that flush to the right edge?


2. tip-manager needs some work:
- The close button should be size M (24px)
- close button should have no padding around it (see hover/press). should be edge to edge
- padding all around needs another pass. see figma: https://www.figma.com/file/E3SB0i5wPy7AagB7KcjF05/?node-id=1183%3A20943
- the < > paginations are using action M, lets change that to scale S.

**Comment by bstifle** [comment]
@justinhough yo, last minute change if you can. Could get the dividing border on tip-manager to use border-3? This will match modal and date-picker to be more conhesive

**Review by bstifle — APPROVED** [review]
Lookin good!!! ✅ ✔️

File: src/components/calcite-tip-manager/calcite-tip-manager.scss [hunk 1]
```diff
@@ -8,10 +8,15 @@
  */
 
 :host {
-  @extend %component-host;
   @apply block;
-  --calcite-tip-manager-height: 18vh;
-  --calcite-tip-max-width: 540px;
+  --calcite-tip-manager-height: 19vh;
+  //--calcite-tip-max-width: 600px;
```
Comment (asangma) [inline src/components/calcite-tip-manager/calcite-tip-manager.scss:13]: cleanup comment.

File: src/components/calcite-tip-manager/calcite-tip-manager.scss [hunk 2]
```diff
@@ -8,10 +8,15 @@
...
+  * {
+    @apply box-border;
+  }
+  background-color: var(--calcite-ui-foreground-1);
+  color: var(--calcite-ui-text-2);
+  font-size: var(--calcite-font-size--1);
```
Comment (asangma) [inline src/components/calcite-tip-manager/calcite-tip-manager.scss:19]: Use `@apply` here.

File: src/components/calcite-tip-manager/calcite-tip-manager.scss [hunk 3]
```diff
@@ -21,15 +26,26 @@
 @import "../../assets/styles/header";
 
 .header .heading {
-  @apply px-2;
+
+}
```
Comment (asangma) [inline src/components/calcite-tip-manager/calcite-tip-manager.scss:30]: Should this have styles in it?

File: src/components/calcite-tip-manager/calcite-tip-manager.scss [hunk 4]
```diff
@@ -21,15 +26,26 @@
...
+.header.header {
```
Comment (asangma) [inline src/components/calcite-tip-manager/calcite-tip-manager.scss:31]: This should probably just be `.header`.

File: src/components/calcite-tip-manager/calcite-tip-manager.scss [hunk 5]
```diff
@@ -38,23 +54,24 @@
...
+  @apply mt-0
     overflow-auto
     flex
     justify-center
     items-start
     focus-base;
   animation-name: none;
   animation-duration: 150ms;
-  animation-timing-function: $easing-function;
+  // animation-timing-function: $easing-function;
```
Comment (asangma) [inline src/components/calcite-tip-manager/calcite-tip-manager.scss:65]: cleanup comment.

File: src/components/calcite-tip-manager/calcite-tip-manager.scss [hunk 6]
```diff
@@ -38,23 +54,24 @@
...
   height: var(--calcite-tip-manager-height);
+  margin-top: 1px;
```
Comment (asangma) [inline src/components/calcite-tip-manager/calcite-tip-manager.scss:67]: Use `@apply` for this.

**Review by asangma — CHANGES_REQUESTED** [review]
Just a few things and an update to the Action scale in pagination.

File: src/components/calcite-tip-manager/calcite-tip-manager.tsx [hunk 7]
```diff
@@ -283,6 +285,7 @@ export class CalciteTipManager {
             icon={ICONS.close}
             onClick={this.hideTipManager}
             text={closeLabel}
+            scale="m"
```
[the remark this answers is not in the document]
Comment (julio8a) [reply src/components/calcite-tip-manager/calcite-tip-manager.tsx:288]: This one is M, did you mean something different?

File: src/components/calcite-tip-manager/calcite-tip-manager.tsx [hunk 8]
```diff
@@ -244,13 +244,15 @@ export class CalciteTipManager {
...
           text={previousLabel}
+          scale="s"
         />
         <span class={CSS.pagePosition}>{`${paginationLabel} ${selectedIndex + 1}/${total}`}</span>
         <calcite-action
           class={CSS.pageNext}
           icon={dir === "ltr" ? ICONS.chevronRight : ICONS.chevronLeft}
           onClick={this.nextClicked}
           text={nextLabel}
+          scale="s"
```
[the remark this answers is not in the document]
Comment (bstifle) [reply src/components/calcite-tip-manager/calcite-tip-manager.tsx:255]: The Action should be M, which means the icon should be scale S

File: src/components/calcite-tip-manager/calcite-tip-manager.tsx (hunk 7 above)
[the remark this answers is not in the document]
Comment (bstifle) [reply src/components/calcite-tip-manager/calcite-tip-manager.tsx:288]: The Action should be M, which means the icon should be scale S

[the remark this answers is not in the document]
Comment (asangma) [reply src/components/calcite-tip-manager/calcite-tip-manager.tsx:288]: My mistake.

File: src/components/calcite-tip/calcite-tip.scss [hunk 9]
```diff
@@ -5,13 +5,11 @@
...
-      @apply box-border;
-    }
-    background-color: var(--calcite-ui-foreground-1);
-    color: var(--calcite-ui-text-2);
-    font-size: var(--calcite-font-size--1);
+    border-color-2
+    box-border
+    bg-foreground-1
+    text-color-2
+    text--1h;
```
Comment (asangma) [inline src/components/calcite-tip/calcite-tip.scss:12]: This is missing the wildcard selector and box-border style from the component-host partial.

File: src/components/calcite-tip-manager/calcite-tip-manager.scss [hunk 10]
```diff
@@ -8,15 +8,8 @@
  */
 
 :host {
-  @apply block;
+  @apply block box-border bg-foreground-1 text-color-2 text--1h;
```
Comment (asangma) [inline src/components/calcite-tip-manager/calcite-tip-manager.scss:11]: nitpick: when there are more than three rules, put styles on individual lines.

**Review by asangma — CHANGES_REQUESTED** [review]
A couple minor things and a formatting nitpick.

**Review by julio8a — APPROVED** [review]

**Pull request merged**